### PR TITLE
Fix `DisposedObjectException` in DatabaseController

### PR DESCRIPTION
**Pull request by seveneleven**
Not awaiting the `setup.Execute` call in `ModelSetupExecutor.Execute` lead to
`DisposedObjectException`

### CODE REVIEW

File: src/Moryx.AbstractionLayer.TestTools/InMemoryUnitOfWorkFactoryBuilder.cs [hunk 1]
```diff
@@ -39,12 +55,13 @@ public static class InMemoryUnitOfWorkFactoryBuilder
         /// <typeparam name="T"></typeparam>
         /// <param name="factory"></param>
         /// <returns></returns>
-        public static UnitOfWorkFactory<T> EnsureDbIsCreated<T>(this UnitOfWorkFactory<T> factory)
```
Comment (1nf0rmagician) [inline src/Moryx.AbstractionLayer.TestTools/InMemoryUnitOfWorkFactoryBuilder.cs:42]: Unfortunately, we publish the Moryx.AbstractionLayer.TestTools project as a package, i.e. this should be a breaking change if I'm not mistaken 🙈

File: src/Moryx.Model.Sqlite/SqliteDbContextManager.cs [hunk 2]
```diff
@@ -85,5 +95,14 @@ public void UpdateConfig(Type dbContextType, Type configuratorType, IDatabaseCon
         {
             throw new NotImplementedException();
         }
+
+        private static ConfigManager CreateConfigManager()
```
Comment (1nf0rmagician) [inline src/Moryx.Model.Sqlite/SqliteDbContextManager.cs:99]: You do not use this method do you? 🤔

File: src/Moryx.Model/ModelSetupExecutor.cs [hunk 3]
```diff
@@ -14,7 +14,7 @@
 namespace Moryx.Model
 {
     /// <inheritdoc />
-    internal class ModelSetupExecutor<TContext> : IModelSetupExecutor
```
Comment (1nf0rmagician) [inline src/Moryx.Model/ModelSetupExecutor.cs:17]: Why did you make this public? 🤔

File: src/Moryx.Runtime.Endpoints/Databases/Endpoint/DatabaseController.cs [hunk 4]
```diff
@@ -227,7 +227,7 @@ public async Task<ActionResult<DatabaseMigrationSummary>> MigrateDatabaseModel(s
 
         [HttpPost("{targetModel}/setup")]
         [Authorize(Policy = RuntimePermissions.DatabaseCanSetup)]
-        public ActionResult<InvocationResponse> ExecuteSetup(string targetModel, ExecuteSetupRequest request)
```
Comment (1nf0rmagician) [inline src/Moryx.Runtime.Endpoints/Databases/Endpoint/DatabaseController.cs:230]: Not 100% sure, is this a breaking change? I don't know how Asp translates this to http 🤔

**Review by 1nf0rmagician — COMMENTED** [review]
An please provide a description for the MR and make the title a normal sentance 😊

File: src/Moryx.Model.Sqlite/SqliteDbContextManager.cs (hunk 2 above)
Comment (seveneleven) [reply src/Moryx.Model.Sqlite/SqliteDbContextManager.cs:99]: Ist used ~10 lines above: 
```
configurator.Initialize(typeof(TContext), CreateConfigManager(), null);
```

File: src/Moryx.Model/ModelSetupExecutor.cs (hunk 3 above)
Comment (seveneleven) [reply src/Moryx.Model/ModelSetupExecutor.cs:17]: I lined that out in the commit:
> * `ModelSetupExecutor` is public to be used by `IDbContextManager`
>   implementations in different packages

File: src/Moryx.Model.Sqlite/SqliteDbContextManager.cs (hunk 2 above)
Comment (1nf0rmagician) [reply src/Moryx.Model.Sqlite/SqliteDbContextManager.cs:99]: Ahh yes, overlooked that one 🙈

**Review by 1nf0rmagician — APPROVED** [review]

**Pull request merged**